pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/MailboxSendOperator.java

```diff
@@ -49,9 +47,6 @@ public class MailboxSendOperator extends BaseOperator<TransferableBlock> {
   private static final Logger LOGGER = LoggerFactory.getLogger(MailboxSendOperator.class);
 
   private static final String EXPLAIN_NAME = "MAILBOX_SEND";
-  private static final Set<RelDistribution.Type> SUPPORTED_EXCHANGE_TYPE =
-      ImmutableSet.of(RelDistribution.Type.SINGLETON, RelDistribution.Type.RANDOM_DISTRIBUTED,
-          RelDistribution.Type.BROADCAST_DISTRIBUTED, RelDistribution.Type.HASH_DISTRIBUTED);
```

Comment:
Broadcast shouldn't be removed imo. Otherwise a non partition / non co-located join will always use a shuffle / hash distributed exchange.

Reply:
Broadcast is only used when join key is empty now after SkipShuffle is disabled as said above